src/foam/u2/view/PopUpTitledView.js

```diff
@@ -0,0 +1,241 @@
+/** 
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'PopUpTitledView',
```

Comment:
Rename PopUp to Popup to be consistent with other uses of the word.

Reply:
Updated